src/edge/models/fragment_updater.py

```diff
@@ -127,12 +127,13 @@ def remove_bases(self, before_base1, length):
 
     def replace_bases(self, before_base1, length_to_remove, sequence):
 
-        if length_to_remove <= 0:
+        if length_to_remove < 0:
             raise Exception("Cannot remove less than one chunk pair")
```

Comment:
Cannot remove fewer than zero chunk paris?

Reply:
+1 on maybe changing that Exception message